core/logger/test_logger_test.go

```diff
@@ -35,8 +37,8 @@ func TestTestLogger(t *testing.T) {
 		testMessage = "Test message"
 	)
 	lgr.Warn(testMessage)
-	// [WARN]  Test message		logger/test_logger_test.go:23    logger=TestLogger
-	requireContains("[WARN]", testMessage, fmt.Sprintf("logger=%s", testName))
+	// [WARN]  Test message		logger/test_logger_test.go:23    logger=1.0.0@sHaValue.TestLogger
+	requireContains("[WARN]", testMessage, fmt.Sprintf("logger=%s@%s.%s", static.Version, static.Sha[:7], testName))
```

Comment:
Maybe should use `baseLoggerName()` for these too? Dedupes and limits the potential slice panic to one location where it's obvious what is wrong :shrug:

Reply:
makes sense, updated.